src/NzbDrone.Core/Indexers/Definitions/TorrentSyndikat.cs

```diff
@@ -141,7 +141,11 @@ private IEnumerable<IndexerRequest> GetPagedRequests(string term, int[] categori
             {
                 queryCollection.Add("imdbId", imdbId);
             }
-            else if (!string.IsNullOrWhiteSpace(searchString))
+            else if (tvdbId != null && (Settings.PreferTvDbId || string.IsNullOrWhiteSpace(searchString)))
```

Comment:
Okay, i think i got it now.

> Basically in this case using the ID only - as I alluded to earlier - would return every release for the series on the site and Sonarr only pages back up to 1,000 results. so for any and all series with more than 1,000 releases on the site when searched by ID....sonarr would never be able to find those releases.

That is indeed the case for a total of 2 shows on this tracker. 

> so having it as an option - unless no results are returned if the option is not selected somehow - isn't feasible.

It depends, since its a german tracker, mainly german titles / release names are used. As i stated earlier in my example there are 0 results when searching for "Married with Children".

In that specific case, for this tracker, i have the feeling that the TVDBId search is the lesser evil because everytime one calls the API with a english title which differs from the german title there are 0 results.